awscli/paramfile.py

```diff
@@ -125,8 +126,18 @@ class ResourceLoadingError(Exception):
 
 def register_uri_param_handler(session, **kwargs):
     prefix_map = copy.deepcopy(LOCAL_PREFIX_MAP)
-    fetch_url = session.get_scoped_config().get(
-        'cli_follow_urlparam', 'true') == 'true'
+    try:
```

Comment:
Yeah I think it would be good if we moved one of the integration tests related to missing profile to be functional tests for this one so we can catch them in PR's.